main.go

```diff
@@ -23,6 +23,20 @@ type Config struct {
 	Workdir    string `env:"workdir"`
 	Command    string `env:"command,required"`
 	NpmVersion string `env:"npm_version"`
+	CacheLevel string `env:"cache_level,opt[none,local,global]"`
+}
+
+func parseCacheLevel(level string) cacheLevel {
+	switch level {
+	case "none":
```

Comment:
set the base type of your new type const to have a string instead of int. so then you no need to have int<->string conversion

Reply:
here: https://github.com/bitrise-steplib/steps-npm/pull/19/files#diff-56ffa6659747ba7a23d2b5597a4cda5dR14